mesonbuild/dependencies/python.py

```diff
@@ -327,6 +327,12 @@ def __init__(self, name: str, environment: 'Environment',
         if not self.link_libpython and mesonlib.version_compare(self.version, '< 3.8'):
             self.link_args = []
 
+        # But not Apple, because it's a framework
+        if self.env.machines.host.is_darwin() and 'PYTHONFRAMEWORKPREFIX' in self.variables:
+            framework_prefix = self.variables['PYTHONFRAMEWORKPREFIX']
+            # Add rpath, will be de-duplicated if necessary
+            if framework_prefix.startswith('/Applications/Xcode.app/'):
+                self.link_args += ['-rpath,' + framework_prefix]
```

Comment:
This should be `-Wl,-rpath,` not `-rpath,`: https://gitlab.freedesktop.org/gstreamer/cerbero/-/jobs/70298085#L2090

@amyspark could you please fix this and also add a test while you're at it? Looks like this code is not tested on CI at all.

Reply:
→ https://github.com/mesonbuild/meson/pull/14211